src/main/java/com/internet/shop/dao/impl/ProductDao.java

```diff
@@ -0,0 +1,17 @@
+package com.internet.shop.dao.impl;
```

Comment:
```suggestion
package com.internet.shop.dao;
```
interfaces should be in `dao` folder
their implementations - in `dao.impl` subfolder

Reply:
done